sideboard/tests/test_configuration.py

```diff
@@ -4,8 +4,8 @@
 import pytest
 from mock import Mock
 
-from sideboard.config import get_config_files, get_config_overrides, \
-    get_config_root, get_module_and_root_dirs, parse_config, uniquify
+from sideboard.lib import config
+from sideboard.config import get_config_files, get_config_overrides, get_config_root, get_module_and_root_dirs, parse_config, uniquify
```

Comment:
If you're ever curious about which lines of code I wrote in one of our codebases, just check which lines adhere to a maximum of 79 characters 😄